### PR TITLE
Added shell script to run it

### PR DESCRIPTION
**Pull request by jbingham**
Added a shell script to run Dockerflow and updated docs.
Also fixed typo in yaml GATK example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,15 +76,20 @@ Run the following steps on your laptop or local workstation:
...
 
-3.  Run a sample workflow:
+3. Set up the DOCKERFLOW_HOME environment.
 
-        java -jar target/dockerflow*dependencies.jar \
-        --project=MY-PROJECT \
-        --workflow-file=src/test/resources/linear-graph.yaml \
-        --workspace=gs://MY-BUCKET/MY-PATH \
-        --runner=DirectPipelineRunner
+        export DOCKERFLOW_HOME=`pwd`
```
Comment (mbookman) [inline README.md]: The `cmd` notation is considered legacy. The public Google bash style guide encourages the use of $(cmd).

https://google.github.io/styleguide/shell.xml?showone=Command_Substitution#Command_Substitution

**Comment by jbingham** [comment]
Thanks! Updated.

File: README.md [hunk 2]
```diff
@@ -76,15 +76,20 @@ Run the following steps on your laptop or local workstation:
...
 
-3.  Run a sample workflow:
+3. Set up the DOCKERFLOW_HOME environment.
 
-        java -jar target/dockerflow*dependencies.jar \
-        --project=MY-PROJECT \
-        --workflow-file=src/test/resources/linear-graph.yaml \
-        --workspace=gs://MY-BUCKET/MY-PATH \
-        --runner=DirectPipelineRunner
+        export DOCKERFLOW_HOME="$(command pwd)"
```
Comment (mbookman) [inline README.md]: Should be just:

    export DOCKERFLOW_HOME="$(pwd)"

File: README.md [hunk 3]
```diff
@@ -76,15 +76,20 @@ Run the following steps on your laptop or local workstation:
...
-3.  Run a sample workflow:
+3. Set up the DOCKERFLOW_HOME environment.
 
-        java -jar target/dockerflow*dependencies.jar \
-        --project=MY-PROJECT \
-        --workflow-file=src/test/resources/linear-graph.yaml \
-        --workspace=gs://MY-BUCKET/MY-PATH \
-        --runner=DirectPipelineRunner
+        export DOCKERFLOW_HOME="$(pwd)"
+        export PATH=$PATH:$DOCKERFLOW_HOME/bin
```
Comment (jbingham) [inline README.md]: Done

Comment (jbingham) [inline README.md]: Done

**Pull request merged**